### DISCUSSION TITLE
Can I limit the number of listed available versions?

### DISCUSSION THREAD

**Discussion by nkiesel** [Splitties/refreshVersions#522] [Q&A]
Hi,
some of our dependencies are on `version.software.amazon.awssdk`, and all these create new versions on a nearly daily basis. We are using a 2 month old version right now, which thus lists 50+ available versions in `version.properties`. Is there any way to limit this (e.g. only show the first 5 and last 5)?

**Comment by jmfayard** [comment]
You can add a rule `rejectVersionIf` 

https://jmfayard.github.io/refreshVersions/update-dependencies/#filter-which-versions-are-added-to-versionsproperties

**Reply by nkiesel** [reply]
thx. I'm trying to figure out what exactly I can use in this `rejectVersionIf`. In my concrete case, I would need access to the total number of available versions and the current versions position in that list, or alternatively the total list of available versions.  I cloned the repo to find the relevant code but I'm still struggling a bit. Can you point me to the relevant code?

**Reply by jmfayard** [reply]
Currently you can't say "only the last 5"

But you can disable this particular version key

```
refreshVersions {
    rejectVersionIf {
        versionKey == "version.software.amazon.awssdk"
    }
}
```
